ml-notebook/binder/postBuild

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+# labextensions
+
+jupyter labextension install dask-labextension@1.0.0 \
```

Comment:
If you want the nvdashboard you'll also need https://www.npmjs.com/package/jupyterlab-nvdashboard